### PR TITLE
[ft-benchmark] benchmarknet

### PR DESCRIPTION
**Pull request by MCJOHN974**

### CODE REVIEW

File: core/primitives-core/src/chains.rs [hunk 1]
```diff
@@ -11,3 +11,6 @@ pub const STATELESSNET: &str = "statelessnet";
 
 /// Pre-release testing environment.
 pub const MOCKNET: &str = "mocknet";
+
+/// Used by ft-benchmark. TODO: Put link to ft-benchmark docs.
```
Comment (akashin) [inline core/primitives-core/src/chains.rs:15]: You can use http://go/crt-benchmark as a link here.

File: core/parameters/src/config_store.rs [hunk 2]
```diff
@@ -136,6 +138,18 @@ impl RuntimeConfigStore {
                 let genesis_runtime_config = RuntimeConfig::initial_testnet_config();
                 Self::new(Some(&genesis_runtime_config))
             }
+            near_primitives_core::chains::BENCHMARKNET => {
+                let mut config_store = Self::new(None);
+                let mut config = RuntimeConfig::clone(config_store.get_config(PROTOCOL_VERSION));
+                config.congestion_control_config.max_tx_gas = 10u64.pow(8);
```
Comment (akashin) [inline core/parameters/src/config_store.rs:144]: I think a good value for max and min gas would be 100 PGas, so higher than the current value.

Comment (akashin) [reply core/parameters/src/config_store.rs:144]: Actually, we can also use an old "unlimited" value: 9_223_372_036_854_775_807

Comment (MCJOHN974) [reply core/parameters/src/config_store.rs:144]: Btw I seen high tps on gcp without changing this values

File: core/parameters/src/config_store.rs [hunk 3]
```diff
@@ -136,6 +138,18 @@ impl RuntimeConfigStore {
                 let genesis_runtime_config = RuntimeConfig::initial_testnet_config();
                 Self::new(Some(&genesis_runtime_config))
             }
+            near_primitives_core::chains::BENCHMARKNET => {
```
Comment (akashin) [inline core/parameters/src/config_store.rs:141]: We need to add a detailed comment here:
```rust
// In benchmarknet, we are measuring the peak throughput that the NEAR network can handle while still being stable.
// This requires increasing the limits below that are set too conservatively.
```

**Review by akashin — APPROVED** [review]

File: core/parameters/src/config_store.rs [hunk 4]
```diff
@@ -130,12 +132,26 @@ impl RuntimeConfigStore {
     /// first protocol versions.
```
[the remark this answers is not in the document]
Comment (MCJOHN974) [reply core/parameters/src/config_store.rs:132]: It says "for first protocol versions" while we are overriding only last.

**Pull request merged**